### PR TITLE
Remove default max from auto configuration

### PR DESCRIPTION
**Pull request by rajnikantsh**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3403)

### CODE REVIEW

File: aws/build.gradle [hunk 1]
```diff
@@ -93,7 +93,6 @@ task deploy(type: AWSLambdaInvokeTask) {
   "experiments": "${ext.getParameterValue('beamExperiments') ?: ext.getParameterValue(getCurrentGitBranch() + '.experiments')}",
   "execute_class": "${ext.getParameterValue('executeClass')}",
   "execute_args": "${ext.getParameterValue('executeArgs')}",
-  "forced_max_ram": "${ext.getParameterValue('forcedMaxRAM')}",
```
Comment (dgm90) [inline aws/build.gradle:96]: why remove this parameter?

Comment (rajnikantsh) [reply aws/build.gradle:96]: this parameter was taking default as 'null' . we don't need to use its value as null. so completely removed it. If someone need this param then they need to provide some valid value

**Review by dgm90 — APPROVED** [review]

**Review by nikolayilyin — APPROVED** [review]
lgtm

**Comment by dgm90** [comment]
Test!

**Pull request merged**